charts/ibm-fhir-server/values.yaml

```
@@ -41,6 +41,32 @@ endpoints:
       - update
       - patch
       - delete
+
+# see <https://github.com/bitnami/charts/tree/master/bitnami/postgresql> for details
+postgresql:
+  # -- enable an included PostgreSQL DB.
+  # if set to `false`, the connection settings under the `db` key are used
+  enabled: true
+  image:
+    # -- update the default Postgres version to 13.4
```

I think this should document the property, not its value.
```suggestion
    # -- the tag for the postgresql image
```